orttraining/orttraining/python/training/ortmodule/__init__.py

```diff
@@ -58,5 +58,20 @@
 from ._custom_autograd_function import enable_custom_autograd_support
 enable_custom_autograd_support()
 
+# Override torch.manual_seed and torch.cuda.manual_seed
+def override_torch_manual_seed(seed):
+    from onnxruntime import set_seed
+    set_seed(seed)
+    return torch_manual_seed(seed)
+torch_manual_seed = torch.manual_seed
+torch.manual_seed = override_torch_manual_seed
```

Comment:
What happens if the user sets the seed on the PyTorch model before importing ORTModule? In that case, we would not be able to align with the set seed, right?

I was looking at the PyTorch docs and came across torch.get_rng_state and torch.set_range_state (https://pytorch.org/docs/stable/torch.html#torch.get_rng_state). Do you think we could leverage that inside the ORTModule constructor by setting the onnxruntime seed with the help of a byte tensor?

Reply:
addressed. 
Thank you for pointing out that torch.initial_seed() is the way to read the seed.